test/admin.test.js

```diff
@@ -6,7 +6,7 @@ const { cloneDeep } = require('lodash')
 const IlpPacket = require('ilp-packet')
 const appHelper = require('./helpers/app')
 const logHelper = require('./helpers/log')
-const logger = require('../src/common/log')
+const logger = require('../dist/common/log')
```

Comment:
Why have the tests `require` modules from `dist` instead of `src`? (In general, not this file in particular).

Reply:
The transpiled JS is only in `dist/` now, so unless the tests were upgraded to TypeScript they can't require anything in `src/`